pytorch_forecasting/data/encoders.py

```diff
@@ -785,7 +785,7 @@ def fit(self, y: pd.Series, X: pd.DataFrame):
             self
         """
         y = self.preprocess(y)
-        eps = np.finfo(np.float).eps
```

Comment:
32 or 64 bits? We cannot use `y.dtype` because it is possible to have different type for y and X.